Documentation/concepts/networking/fragmentation.rst

```diff
@@ -21,4 +21,11 @@ options:
 - ``--bpf-fragments-map-max``: Control the maximum number of active concurrent
   connections using IP fragmentation. For the defaults, see `bpf_map_limitations`.
 
+.. note::
+
+    When running Cilium with kube-proxy, even nodePort traffic is handled by
+    kube-proxy, IPv4 fragmented packets can still break due to a kernel bug where
+    route MTU is not respected for forwarded packets. It has been (`fixed <http://patchwork.ozlabs.org/project/netdev/patch/20200923201815.388347-1-zenczykowski@gmail.com/>`_)
```

Comment:
```suggestion
    route MTU is not respected for forwarded packets. It has been `fixed <https://patchwork.ozlabs.org/project/netdev/patch/20200923201815.388347-1-zenczykowski@gmail.com/>`_
```

Reply:
Maybe we could link to [the upstream commit](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=02a1b175b0e92d9e0fa5df3957ade8d733ceb6a0) instead?

Reply:
done. switched to the upstream commit link.